README.md

```diff
@@ -427,6 +433,30 @@ const uiSchema = {
 };
 ```
 
+#### `addable` option
+
+if either `items` or `additionalItems` contains a schema object, an add button for new items is shown by default. You can turn this off with the `addable` option in `uiSchema`:
```

Comment:
Thank you for your reply. I knew about fixed arrays, I even added a fixed sized array example with 2 unremovable additionalItems in the playground. :)

The use case I have, is that the items _are_ addable, reorderable and removable, but not via _this_ form. Another user may add items in another form, but this user may not add items in this form. So the items in the array are really part of the data, not part of the schema. If I'm not mistaken, a fixed array in the schema would not be a right representation of the data. Then every instance of this particular data type should have their own schema, which would undermine the purpose of json schema.

In angular-schema-form, which appears to be maintained by the same organisation as JSON Schema, the [api for this](https://github.com/json-schema-form/angular-schema-form/blob/development/docs/index.md#array) is as follows (a `uiSchema` is called a `form` in angular-schema-form):

> To suppress add and remove buttons set add to null and remove to null.

```js
function FormCtrl($scope) {
  $scope.form = [
    {
      key: "subforms",
      add: null,
      remove: null,
      style: {
        add: "btn-success"
      },
      items: [
        "subforms[].nick",
        "subforms[].name",
        "subforms[].emails",
      ],
    }
  ];
}
```